commands/operator-sdk/cmd/test.go

```diff
@@ -31,10 +32,9 @@ var (
 	verbose          bool
 )
 
-// TODO: allow users to pass flags through to `go test`
 func NewTestCmd() *cobra.Command {
 	testCmd := &cobra.Command{
-		Use:   "test --test-location <path to tests directory> [flags]",
+		Use:   "test --test-location <path to tests directory> [flags] args [go test flags]",
```

Comment:
Can I do `operator-sdk test --test-location=e2e -c=deploy/crd.yaml -o=deploy/operator.yaml -c -o=e2e.test`? Where the latter `-c` and `-o` are the go test flags.

Reply:
Yes this would work fine

Reply:
Yeah, there's 2 ways of handling this right now. We either use `args` as a separator between the go test args and the sdk's args or we ask people to put all of their args into a string (so we would have something like `operator-sdk test -t ./test/e2e --passthrough "-v -parallel 2"`).
@hasbro17 what do you think would be the better choice for this?

Reply:
I vote for using the passthrough flag for a potential better user experience and the consistency with `operator-sdk up local --operator-flags`.

Reply:
Yeah we can use an explicit flag to pass on the go test flags. Lets call it `--go-test-flags` instead of `--passthrough` though to be clear.